parity/account_utils.rs

```diff
@@ -81,7 +81,7 @@ mod accounts {
 		let account_settings = AccountProviderSettings {
 			unlock_keep_secret: cfg.enable_fast_unlock,
 			blacklisted_accounts: 	match *spec {
```

Comment:
For my education, what is this blacklist for? The docs are oxymoronic: "/// Disallowed accounts.". What did `00a329c0648769a73afac7f9381e08fb43dbea72`do to become blacklisted?

Reply:
It is a relict from the parity wallet UI. there was a chance that a user did not follow instructions and recovered an account from an empty seed phrase. this account is blacklisted from mainnet usage for safety reasons. but as we no longer have the wallet, we could also remove this.